packages/test/test-end-to-end-tests/src/test/dataStoresNested.spec.ts

```diff
@@ -156,26 +148,43 @@ describeCompat("Nested DataStores", "NoCompat", (getTestObjectProvider, apis) =>
 		// Have a second container that follows passively the first one
 		await addContainerInstance();
 		await provider.ensureSynchronized();
-		return dataStores;
+		return channelCollection;
 	}
 
 	it("Basic test", async () => {
-		const dataStores = await initialize();
+		const channelCollection1 = await initialize();
 
-		const context = dataStores.createDataStoreContext([testObjectFactory.type]);
+		const context = channelCollection1.createDataStoreContext([testObjectFactory.type]);
 		const url = `/${context.id}`;
 		const channel = await context.realize();
 		channel.makeVisibleAndAttachGraph();
-		const testObject1 = (await dataStores.request({ url })).value as TestFluidObject;
+		const testObject1 = (await channelCollection1.request({ url })).value as TestFluidObject;
 		testObject1.root.set("testKey", 100);
 
 		await waitForSummary();
-		const dataStores2 = (await addContainerInstance()).dataStores;
+		const channelCollection2 = (await addContainerInstance()).channelCollection;
+
+		await provider.ensureSynchronized();
+
+		const testObject2 = (await channelCollection2.request({ url })).value as TestFluidObject;
+		assert.strictEqual(testObject2.root.get("testKey"), 100, "same value");
+	});
+
+	it("Aliasing", async () => {
+		const channelCollection1 = await initialize();
+		const dataStore1 = await channelCollection1.createDataStore([testObjectFactory.type]);
+		assert.equal(await dataStore1.trySetAlias("test"), "Success");
```

Comment:
How does it work? I do not think we have any code to serialize alias map for nested channels yet, no?

Reply:
It does not (see the last line). The goal is to make it work in the next PR.